.github/workflows/case_studies.yml

```diff
@@ -39,29 +39,23 @@ jobs:
         with:
           lfs: true
 
-      - name: Checkout LFS objects
-        run: git lfs checkout
+      - name: Install poetry
+        run: pipx install poetry
```

Comment:
Switching to `pipx` here seems to have broken the night case studies job. Error message:
```
Error: Unable to locate executable file: poetry. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also check the file mode to verify the file is executable.
```

Reply:
This is the error that is raised after I installed `pipx` (via `apt`) on the server. I could be that the version of pipx installed is installing poetry in the wrong path. But it may be simpler just to avoid using pipx.

Reply:
ah good point, I forgot that one is ran on the server. I will fix it